src/main/java/carRacing/RacingCarApplication.java

```diff
@@ -0,0 +1,11 @@
+package carRacing;
+
+import carRacing.controller.RacingCarGameManager;
+
+public class RacingCarApplication {
+    public static void main(String[] args) {
+        RacingCarGameManager game = new RacingCarGameManager();
+        game.run();
```

Comment:
```suggestion
        RacingCarGameManager.run();
```

인스턴스 변수가 없는(즉, 상태가 없는) 객체는 굳이 인스턴스를 생성할 필요가 없다.
굳이 인스턴스를 생성할 필요가 없는 경우 클래스 메서드로 구현해보면 어떨까?